### PR TITLE
wip test fixes 27.11.2

### PR DESCRIPTION
**Pull request by orndorffgrant**

### CODE REVIEW

File: features/docker.feature [hunk 1]
```diff
@@ -17,7 +17,7 @@ Feature: Build docker images with pro services
 
         RUN --mount=type=secret,id=ua-attach-config \
             apt-get update \
-            && apt-get install --no-install-recommends -y ubuntu-advantage-tools ca-certificates \
+            && env DEBIAN_FRONTEND=noninteractive apt-get install -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" --no-install-recommends -y ubuntu-advantage-tools ca-certificates \
```
Comment (renanrodrigo) [inline features/docker.feature:20]: this change is not needed; there may be a bug in the `| true` below though

Comment (orndorffgrant) [reply features/docker.feature:20]: oops yeah I didn't even mean to push that, will fix

**Comment by renanrodrigo** [comment]
@lucasmoura we need @blackboxsw to update the secrets for this repo with the new production token

**Comment by renanrodrigo** [comment]
(re)sent a commit with the change I did to run the docker tests without failures

**Review by lucasmoura — APPROVED** [review]

**Pull request merged**